comms-publish-service/orb.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
+description: "Orb for deploying a service (adapted to the Comms team only)"
+
+executors:
+  docker-executor:
+    docker:
+      - image: circleci/openjdk:14.0.2-jdk-buster
+
+commands:
+  generate_sbt_cache_key:
+    description: "Generate sbt cache key"
+    steps:
+      - run: md5sum project/build.properties project/plugins.sbt build.sbt > .sbt_cache_key
+
+  restore_sbt_cache:
+    description: "Restore sbt cache"
+    steps:
+      - generate_sbt_cache_key
+      - restore_cache:
+          keys:
+            - sbt-{{ checksum ".sbt_cache_key" }}
+            - sbt
+
+jobs:
+  publish_service:
```

Comment:
I like the concept of this orb. Are there other jobs we could add to it like build, unit test, integration test, etc?

Reply:
Definitely, I was planning to do that later when I know that this works